pilot/pkg/networking/core/v1alpha3/route/retry/retry.go

```diff
@@ -83,6 +94,30 @@ func ConvertPolicy(in *networking.HTTPRetry) *route.RetryPolicy {
 	if in.PerTryTimeout != nil {
 		out.PerTryTimeout = util.GogoDurationToDuration(in.PerTryTimeout)
 	}
+
+	if in.RetryRemoteLocalities != nil && in.RetryRemoteLocalities.GetValue() {
+		// UpdateFrequency indicated how often to update the priority.
+		// for now we set a default value: 2 for it.
+		previousPriorities := &previouspriorities.PreviousPrioritiesConfig{
+			UpdateFrequency: int32(2),
+		}
+
+		data, err := proto.Marshal(previousPriorities)
+		if err != nil {
+			log.Errorf("error marshal previousPriorities: %v", err)
+		} else {
+			out.RetryPriority = &route.RetryPolicy_RetryPriority{
```

Comment:
Can we make this a constant instead of marshalling this over and over again? since it never changes

Reply:
Done